053-SentinelAutomatedResponse/Coach/Solution-02.md

```diff
@@ -0,0 +1,82 @@
+# Challenge-02 - Custom Queries & Watchlists - Coach's Guide
+
+[< Previous Challenge](./Solution-01.md) - **[Home](./README.md)** - [Next Challenge>](./Solution-03.md)
+
+# Custom Queries & Watchlists
+## Introduction
+In this section of the Coach's guide, you fill find guidance, code, and examples that will help you guide the teams over the course of the WTH. 
+In the spirit of continuous improvement, update this repository with any suggestions, altertnatives, or additional challenges.
```

Comment:
Same comments as the Challenge 1 Coach guide.  Include a link to the WTH contribution guide.

https://aka.ms/wthcontribute